unit_test/test_spdm_responder/finish.c

```diff
@@ -3724,7 +3845,7 @@ int libspdm_responder_finish_test_main(void)
     const struct CMUnitTest spdm_responder_finish_tests[] = {
         /* Success Case*/
         cmocka_unit_test(libspdm_test_responder_finish_case1),
-        /* Can be populated with new test.*/
+        /* The requester and responder have not set HANDSHAKE_IN_THE_CLEAR*/
         cmocka_unit_test(libspdm_test_responder_finish_case2),
```

Comment:
`libspdm_test_responder_finish_case2` will run nothing, what's the meaning here?